XIVLauncher/Accounts/XivAccount.cs

```diff
@@ -51,6 +51,40 @@ public string Password
                 Log.Information($"Set Password SaveCredentials: {b}");
             }
         }
+        
+        [JsonIgnore]
+        public string OtpUri
+        {
+            get
+            {
+                var credentials = CredentialManager.GetCredentials($"FINAL FANTASY XIV-{UserName.ToLower()}-OTP");
+
+                return credentials != null ? credentials.Password : string.Empty;
```

Comment:
I'm curious: how long does this value stick around? I'd be more comfortable using this feature if I knew it was only kept in memory for strictly as long as necessary. Would it be possible to never have a `get` for the URI value, and just have a `get` for the current OTP itself instead?

Reply:
The OTP URI is only kept in memory when it is used to generate your 2FA code on Login OR when on its Setup screen. Otherwise it should never be loaded. Of course you do have to worry about the GC to actually remove it from memory, but it shouldn't be in memory for an inordinate amount of time.